lte/gateway/c/oai/tasks/amf/Registration.cpp

```diff
@@ -762,7 +764,7 @@ int amf_proc_registration_complete(
               ue_amf_context->amf_context.amf_procedures->amf_specific_proc;
       amf_ctx = &ue_amf_context->amf_context;
 
-      stop_timer(&amf_app_task_zmq_ctx, registration_proc->T3550.id);
+      amf_app_stop_timer(registration_proc->T3550.id);
       OAILOG_INFO(
           LOG_AMF_APP, "Timer: after stop registration timer with id = %d\n",
```

Comment:
here id is mentioned good. in addition to ID, can you indicate Timer T3xxx in the log statement?